protos/local/core_interface.proto

```diff
@@ -116,19 +117,42 @@ message SignalWorkflow {
 }
 
 message StartActivity {
-    // TODO: add attributes
+    string workflow_namespace = 1;
+    temporal.api.common.v1.WorkflowType workflow_type = 2;
+    temporal.api.common.v1.WorkflowExecution workflow_execution = 3;
+    temporal.api.common.v1.ActivityType activity_type = 4;
+    temporal.api.common.v1.Header header = 5;
+    temporal.api.common.v1.Payloads input = 6;
+    temporal.api.common.v1.Payloads heartbeat_details = 7;
+    google.protobuf.Timestamp scheduled_time = 8 [(gogoproto.stdtime) = true];
+    google.protobuf.Timestamp current_attempt_scheduled_time = 9 [(gogoproto.stdtime) = true];
+    google.protobuf.Timestamp started_time = 10 [(gogoproto.stdtime) = true];
+    int32 attempt = 11;
+    // (-- api-linter: core::0140::prepositions=disabled
+    //     aip.dev/not-precedent: "to" is used to indicate interval. --)
+    google.protobuf.Duration schedule_to_close_timeout = 12 [(gogoproto.stdduration) = true];
+    // (-- api-linter: core::0140::prepositions=disabled
+    //     aip.dev/not-precedent: "to" is used to indicate interval. --)
+    google.protobuf.Duration start_to_close_timeout = 13 [(gogoproto.stdduration) = true];
+    google.protobuf.Duration heartbeat_timeout = 14 [(gogoproto.stdduration) = true];
+    // This is an actual retry policy the service uses.
+    // It can be different from the one provided (or not) during activity scheduling
+    // as the service can override the provided one in case its values are not specified
+    // or exceed configured system limits.
+    temporal.api.common.v1.RetryPolicy retry_policy = 15;
```

Comment:
Can we start instead we just the things you know you need? We may very well end up adding back all of this but we can at least not have these api linter comments etc.

I'd rather start small and add as needed than end up stuck with a bunch of cruft.